tooling/scripts/download-publisher-cli.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/JetBrains/qodana-cli/v2025/platform/utils"
+)
+
+func main() {
```

Comment:
Also let's unify the  config_loader_cli.go and publisher_cli.go jar obtaining, currently one is done using shell script, and the other is go script

Reply:
Implemented